source/modules/max7c0/ublox7rx.h

```diff
@@ -0,0 +1,32 @@
+#ifndef UBLOX7RX
+#define UBLOX7RX 
+
+#include<stdio.h>
+#include<string.h>
+#include<stdint.h>
+
+#define PUBX_RATE_DEFAULT 
```

Comment:
what is this for?

Reply:
I'm going to merge for now as this #define is never used, if this needs to be set for the user's reference (e.g. as a parameter to _rate_command_) feel free to set it and add to the main branch